.github/workflows/release-image.yml

```
@@ -0,0 +1,42 @@
+        type: string
+
+permissions:
+  actions: write
+  contents: read
+
+jobs:
+  build-release-image:
+    runs-on: ubuntu-24.04
+    steps:
+      - name: Set up Go 1.23
+        uses: actions/setup-go@v3
```

actionlint (1.7.7)

the runner of "actions/setup-go@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)